substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/ConditionalConfigurationRegistry.java

```diff
@@ -53,7 +53,9 @@ protected void registerConditionalConfiguration(ConfigurationCondition condition
     public void flushConditionalConfiguration(Feature.BeforeAnalysisAccess b) {
         for (Map.Entry<String, Collection<Runnable>> reachabilityEntry : pendingReachabilityHandlers.entrySet()) {
             TypeResult<Class<?>> typeResult = ((FeatureImpl.BeforeAnalysisAccessImpl) b).getImageClassLoader().findClass(reachabilityEntry.getKey());
-            b.registerReachabilityHandler(access -> reachabilityEntry.getValue().forEach(Runnable::run), typeResult.get());
+            if (typeResult.isPresent()) {
```

Comment:
Could you please explain why this check is now necessary?

Reply:
Testing this PR brought forward the fact that if a class throwing a linkage error when resolved is added to the list of reachability handlers, the build fails since `typeResult.get()` returns `null`.

Reply:
I see, thank you for the clarification. I wonder if it's worth printing a warning to notify the user that some reachability handlers got ignored (ideally with some info about the `LinkafeError`, but that won't be trivial I guess).

FTR in the main repo this is fixed by https://github.com/oracle/graal/commit/fff140bbe181fab684e36a98b554224f7886c6f1

Reply:
Yes, I can add a warning here in the follow-up PR which will add the warning for ignored proxy registrations. The `LinkageError` should be part of the `TypeResult`, so it should be possible to add it to the warning.